installation_tips/check_your_install.py

```diff
@@ -13,15 +16,20 @@ def check_import_si_full():
 
 def _create_recording():
     import spikeinterface.full as si
-    rec, sorting = si.toy_example(num_segments=1, duration=200, seed=1, num_channels=16, num_columns=2)
-    rec.save(folder='./toy_example_recording')
+    rec, sorting = si.generate_ground_truth_recording(
+        durations=[200.],
+        sampling_frequency=30_000.,
+        num_channels=16,
+        num_units=10,
+        seed=2205
+    )
+    rec.save(folder='./toy_example_recording', **job_kwargs)
 
 
 def _run_one_sorter_and_analyzer(sorter_name):
-    job_kwargs = dict(n_jobs=-1, progress_bar=True, chunk_duration="1s")
     import spikeinterface.full as si
     recording = si.load_extractor('./toy_example_recording')
-    sorting = si.run_sorter(sorter_name, recording, output_folder=f'./sorter_with_{sorter_name}', verbose=False)
+    sorting = si.run_sorter(sorter_name, recording, folder=f'./sorter_with_{sorter_name}', verbose=False)
```

Comment:
Should we set n_jobs here? It might be a bit annoying during the install check that users see a million warnings about "n_jobs" not set.